services/device-registry/src/main/java/org/eclipse/hono/deviceregistry/FileBasedTenantsConfigProperties.java

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 1.0 which is available at
+ * https://www.eclipse.org/legal/epl-v10.html
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ */
+
+package org.eclipse.hono.deviceregistry;
+
+import org.eclipse.hono.service.tenant.TenantService;
+import io.vertx.core.Handler;
+import io.vertx.core.json.JsonObject;
+
+
+/**
+ * Configuration properties for the Hono's tenant manager as own server.
+ *
+ */
+public final class FileBasedTenantsConfigProperties {
+
+    private static final String DEFAULT_TENANTS_FILENAME = "/var/lib/hono/device-registry/tenants.json";
+
+    // the name of the file used to persist the tenant content
+    private String filename = DEFAULT_TENANTS_FILENAME;
+    private boolean saveToFile = false;
```

Comment:
what about extracting common properties to a super class extended by all the registry's service implementation config properties classes?

Reply:
Already on the list :+1: 
But not in this PR I thought.